open-metadata-conformance-suite/open-metadata-conformance-suite-server/src/main/java/org/odpi/openmetadata/conformance/tests/repository/instances/TestSupportedRelationshipPropertySearch.java

```diff
@@ -264,22 +264,22 @@ protected void run(TestPhase phase) throws Exception
                 RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getRequirementId());
 
         String end1TypeDefName = relationshipDef.getEndDef1().getEntityType().getName();
+        EntityDef knownEnd1EntityDef = (EntityDef) repositoryHelper.getTypeDefByName(workPad.getLocalServerUserId(), end1TypeDefName);
         EntityDef end1EntityDef = entityDefs.get(end1TypeDefName);
```

Comment:
This line seems to be the problematic one -- if the `end1TypeDefName` is for example `Asset` then the line before it will work fine (drawing from the open type definitions themselves) but this line (drawing from the entities that are implemented in a connector that does not implement the more abstract supertypes) will simply return `null` and the assertion on the very next line will therefore always fail since `end1EntityDef == null`.